dist/challenge-templates/pwn/challenge/Dockerfile

```diff
@@ -13,12 +13,13 @@
 # limitations under the License.
 FROM ubuntu:24.04 as chroot
 
-RUN /usr/sbin/useradd --no-create-home -u 1000 user
```

Comment:
Why?

Reply:
New ubuntu includes the `ubuntu` user by default (what I meant in the comment), which has uid 1000, which makes it taken, so a `useradd -u 1000` will fail, so we have to remove `ubuntu` first.
Alternatively it could stay and we'd run the challenge as `ubuntu` instead of `user`, but I didn't want to ensure nothing depended on the username.